capstone.html

```diff
@@ -0,0 +1,153 @@
+                    <div class="images">
+                        <img src="images/tr-line.jpg" alt="">
+                    </div>
+                    <div class="titles">
+                        <h3>substation construction</h3>
+                    </div>
+                    <div class="introductions">
+                        <p> Construction of a new substation requires a large amount of materials, equipment and labour. You may hear or notice some construction activities depending on your proximity to the substation site.</p>
+                    </div>              
+                </div>
+                <div class="substation-design">
+                    <div class="images">
+                        <img src="images/wind-power.png" alt="">
+                    </div>
+                    <div class="titles">
+                        <h3>generation</h3>
+                    </div>
+                    <div class="introductions">
+                        <p>A power generation plant is a facility designed to produce electric energy from another form of energy, such as:Potential energy from falling water in a hydroelectric facility,Wind energy,Solar electric from solar (photovoltaic) cells,Heat (thermal) energy </p>
+                    </div>              
+                </div>
+                <div class="substation-design">
+                    <div class="images">
+                        <img src="images/power-control4.png" alt="">
+                    </div>
+                    <div class="titles">
+                        <h3>protection and control systems</h3>
+                    </div>
+                    <div class="introductions">
+                        <p>The task of protection and control in substations and in power grids is the provision of all the technical means and facilities necessary for the optimal supervision, protection, control and management of all system components and equipment in high and medium-voltage power systems.</p>
+                    </div>             
+                </div>
+                <div class="substation-design">
+                    <div class="images">
+                        <img src="images/solar-system.png" alt="">
+                    </div>
+                    <div class="titles">
+                        <h3>Solar power system</h3>
+                    </div>
+                    <div class="introductions">
+                        <p>Solar power is energy from the sun that is converted into thermal or electrical energy. Solar energy is the cleanest and most abundant renewable energy source available, and the U.S. has some of the richest solar resources in the world.</p>
+                    </div>           
+                </div>
+            </div>  
+            <div class="see">
+                <a href="#" class="more">see the whole program</a>
+            </div>  
+        </div>
+        <section class="two">
+        </section>
```

Comment:
- [ ] As requested by the previous code reviewer, please  kindly center your program training header . It is currently not at the center. See screenshot below and kindly fix that.
![hafTrainers](https://user-images.githubusercontent.com/62849872/182246192-d74422cd-9f61-4daa-8eec-53718b942979.png)